libnavui-androidauto/src/main/java/com/mapbox/androidauto/car/MainCarContext.kt

```diff
@@ -37,6 +38,8 @@ class MainCarContext(
         MapboxManeuverApi(distanceFormatter)
     }
 
+    val routeAlternativesEnabled = MutableStateFlow(value = true)
```

Comment:
Alternative is to put this mutable state inside the `CarRouteLine`.

we could go either way, but I think this is good here considering this is where similar values are for drop-in-ui
https://github.com/mapbox/mapbox-navigation-android/blob/73f0ea7e2177d41eaad13ec4f80f24199a39146d/libnavui-dropin/src/main/java/com/mapbox/navigation/dropin/NavigationViewContext.kt#L65